src/org/zaproxy/zap/extension/pscanrules/TestInfoPrivateAddressDisclosure.java

```diff
@@ -1,153 +1,159 @@
-/*

```

Comment:
Might be because of:
https://www.kernel.org/pub/software/scm/git/docs/gitattributes.html#_end_of_line_conversion